src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -74,7 +94,9 @@ public LocalDate addWeeks(LocalDate localDate, Integer numberOfWeeks) {
      * - "someDate is today" - если someDate - сегодня
      */
     public String beforeOrAfter(LocalDate someDate) {
-        return someDate + "is today";
+        return someDate.isEqual(LocalDate.now()) ? someDate + " is today"
+                : (someDate.isAfter(LocalDate.now()) ? someDate + " is after " + LocalDate.now()
```

Comment:
since you're using LocalDate.now() three times here, let's make it as local variable

Reply:
Done